README.md

```diff
@@ -2,4 +2,5 @@ Konfetti
 ========
 Konfetti is a little screensaver which draws (semi-transparent) confetti on your desktop.
 
-Available on HaikuWare: http://haikuware.com/directory/view-details/utilities/screensavers/konfetti
+## Screenshot
```

Comment:
I'd say it's obvious that what follows is a screenshot :)
Let's remove his heading and put a ":" at the end of the previous line.